src/databricks/labs/ucx/install.py

```diff
@@ -265,8 +265,9 @@ def warehouse_type(_):
             spark_conf=spark_conf_dict,
             policy_id=policy_id,
         )
-        ws_file_url = self._installation.save(config)
-        if self._prompts.confirm("Open config file in the browser and continue installing?"):
+        ws_file = self._installation.save(config)
+        ws_file_url = self._installation.workspace_link(ws_file.replace(self._installation.install_folder(), ""))
```

Comment:
why do you do that? why not `self._installation.workspace_link('config.yml')` ?

Reply:
Yeah that was irrelevant, although I prefer to use config.__file__ instead in case we change the conf name for whatever reason